crates/cast/src/helpers/constants.rs

```diff
@@ -15,3 +15,6 @@ inputs = []
 pub const UDC_ADDRESS: &str = "0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02bf";
 pub const OZ_CLASS_HASH: &str =
     "0x058d97f7d76e78f44905cc30cb65b91ea49a4b908a76703c54197bca90f81773";
+// used in wait_for_tx. Txs are fetched every 5 seconds
+#[allow(dead_code)]
+pub const DEFAULT_RETRIES: u8 = 12;
```

Comment:
That was the only way I found to resolve the clippy's warning. Please add your suggestions if you have some 🙏 

```
warning: constant `DEFAULT_RETRIES` is never used
  --> crates/cast/src/helpers/constants.rs:19:11
   |
19 | pub const DEFAULT_RETRIES: u8 = 12;
   |           ^^^^^^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default
```